### DISCUSSION TITLE
Apple2GS

### DISCUSSION THREAD

**Discussion by symbios24** [Exarkuniv/RetroPie-Extra#218] [Ideas]
Hello

Is it possible to add this Apple2GS emulator to your script?

git clone https://github.com/digarok/gsplus.git

Depends install:

apt get re2c libsdl2-dev libsdl2-image-dev libfreetype6-dev libpcap0.8-dev

Build:

mkdir build
cd build
cmake
make

Thanks

**Comment by Exarkuniv** [comment]
added
